### PR TITLE
docs: specify case of hex-encoded string

### PR DESCRIPTION
**Pull request by lucab**
This updates the `random_bytes.hex` docs in order to specify that the hex-encoded string uses the lowercase alphabet. It also adds a note about the fixed size of the result.

### CODE REVIEW

File: docs/resources/bytes.md [hunk 1]
```diff
@@ -37,8 +37,8 @@ resource "azurerm_key_vault_secret" "jwt_secret" {
 
 ### Read-Only
 
-- `base64` (String, Sensitive) The generated bytes presented in base64 string format.
-- `hex` (String, Sensitive) The generated bytes presented in hex string format.
+- `base64` (String, Sensitive) The generated bytes presented in base64 format.
+- `hex` (String, Sensitive) The generated bytes presented in lowercase hex format. The length of the encoded string is exactly twice the `length` parameter.
```
Comment (bendbennett) [inline docs/resources/bytes.md:41]:
```suggestion
- `hex` (String, Sensitive) The generated bytes presented in lowercase hexadecimal format. The length of the encoded string is exactly twice the `length` parameter.
```

**Review by bendbennett — COMMENTED** [review]
Hi @lucab 👋 

Thank you for your PR.
I've made one small suggestion on the wording.

**Comment by lucab** [comment]
@bendbennett ack thanks, directly committed your suggestion.

**Review by bendbennett — COMMENTED** [review]
Hi @lucab 

I forgot to mention that'll you need to make the changes to the wording on the `Schema` and then run `make generate` to recreate the docs.

**Comment by lucab** [comment]
@bendbennett my bad, I didn't realize the docpage was autogenerate, I totally missed the very clear header, apologies.
I've now reworked and rebased the PR with code changes, then regenerate the docs.

**Review by bendbennett — APPROVED** [review]
LGTM 🚀 

Thank you for the contribution @lucab

**Pull request merged**